Example/GiniVision/AppCoordinator.swift

```diff
@@ -76,11 +76,14 @@ final class AppCoordinator: Coordinator {
         popToRootViewControllerIfNeeded()
         
         // 3. Validate document
-        do {
-            try document?.validate()
-            showOpenWithSwitchDialog(forDocuments: [document!])
-        } catch {
-            showExternalDocumentNotValidDialog()
+        if let document = document {
+            do {
+                try GiniVisionDocumentValidator.validate(document,
```

Comment:
We have the `GiniVision` class. I would add the validate method to that one instead. It's easier for clients when they have a few classes for interacting with the library. Also it's easier to find methods with fewer classes to check.

Reply:
Makes sense. I will keep the class for the validation, but I will link the `GiniVision` class to it.